### DISCUSSION TITLE
登上绳网了家人们！！

### DISCUSSION THREAD

**Discussion by KeplerZX** [share121/inter-knot#2519] [其他]
![screenshot-1723875592432](https://github.com/user-attachments/assets/a16b2342-b3f4-4020-bd55-97a415a4f45d)

为了登绳网专门建的GitHub账号，请多多关照，新艾利都精英绳匠-法厄南同是也，UID：10126001
!

**Comment by LinkboyTvT** [comment]
现在的绳网新人直接就是精英绳匠了吗，看来绳网不在是当年的绳网了啊（doge）

**Reply by KeplerZX** [reply]
戳辣。英精绳匠来的

**Comment by huiye123-Hub** [comment]
法厄男同是吧（大嘘）
